.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    commonjs: true,
+    es6: true,
+    node: true,
+    mocha: true
+  },
+  extends: [
+    'standard'
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly'
+  },
+  parserOptions: {
+    ecmaVersion: 2018
+  },
+  rules: {
```

Comment:
consider adding `"indent": ["error", 4]` to keep consistency across projects (I'm planning to go for 2 spaces, but let's do it once we have every repo with `eslint`)